src/hooks/useBasisForm/validation/validateInput.ts

```diff
@@ -0,0 +1,23 @@
+const NUMERIC_REGEX = /^\d*$/;
+const DECIMAL_REGEX = /^\d*(\.\d{2})?$/;
+
+export const validateInput = (
+  value: string,
+  inputProps: Record<string, any>
+) => {
+  const { optional, variant } = inputProps;
+
+  if (!value && !optional) {
```

Comment:
If someone accidentally presses Space in the input, it will pass validation, which is not ideal. By looking at the field, it's clear that the value is missing.
Not how in the current implementation we `trim()` the value.
Can we keep this behavior?